src/main/java/org/ow2/petals/deploymentmodel/busmodel/ContainerInstance.java

```diff
@@ -18,7 +36,7 @@
      * JMX port of the current Petals ESB container. A deployment property can be used. Each container (from topology
      * model) without a default JMX port must have one defined in this model.
      */
-    private int jmxPort;
+    private Integer jmxPort;
```

Comment:
Can you explain the meaning of the `null` value ?

Reply:
When `jmxPort` is null, the default value (of reference Container) is checked instead.

Reply:
Ok. Perhaps can you change the javadoc for something as `If {@code null}, the JMX port default value of the container defined at topology model will be used. This default value is never {@code null}, see {@link Container#defaultJmxPort}.`